src/pdbpp.py

```diff
@@ -1646,6 +1664,21 @@ def get_terminal_size():
             # Work around above returning width, height = 0, 0 in Emacs
             width = width if width != 0 else fallback[0]
             height = height if height != 0 else fallback[1]
+
+            # Hack to remove the linewrap. Not sure why it happens...
+            # During a sticky, the current line looks like so (vertical bars
+            # denote the edge of the cmd/powershell window):
+            #   | 2         import pdb; pdb.set_trace()     |
+            #   | 3  ->     a = 1                           |
+            #   |                                           |
+            #   | 4         b = 2                           |
+            # Simply resizing the terminal a little wider results in:
+            #   | 2         import pdb; pdb.set_trace()      |
+            #   | 3  ->     a = 1                            |
+            #   | 4         b = 2                            |
+            # So I figure we just fake the terminal size a bit. /shrug.
+            if has_colorama_on_windows:
+                width -= 1
```

Comment:
Looks like it is related to current line (handling) then?  https://github.com/pdbpp/pdbpp/blob/cb56e1b6126bb87597684d01af6f18c873be85ee/src/pdbpp.py#L1102-L1115
Can you try without `current_line_color`?

Reply:
Looks like this was a holdover of some other, now-unused changes and it looks like it's no longer needed. I've simply reverted it.